environment.rb

```diff
@@ -0,0 +1,4 @@
+paths = Dir["lib/**/*.rb"].reverse
```

Comment:
I would name this file `console.rb` since we're only using it for the console for now